src/App.tsx

```diff
@@ -1,61 +1,138 @@
+import { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './Types/User';
+import { ToDo } from './Types/ToDo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+export function findUser(userId: number): User | null {
+  return usersFromServer.find(user => (
+    user.id === userId
+  )) || null;
+}
+
+export const newToDos: ToDo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: findUser(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setToDo] = useState(newToDos);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState<string>('Choose a user');
```

Comment:
Here you are setting type for the selectedUser variable, but you are not setting it for all the others. Please, be consistent, either set it too all or to nothing